src/fparser/two/utils.py

```diff
@@ -655,21 +655,17 @@ class SequenceBase(Base):
 
     '''
     @staticmethod
-    def match(separator, subcls, string, keep_empty=True):
-        '''Match one or more 'subcls' fparser2 rules separated by 'separator'
-        in string 'string'. If separator is a string then the
-        separator is used to split the input string. If it is a tuple
-        then a regular expression in the second entry of the tuple is
-        used to split the input string and the the first argument is
-        returned as the separator.
+    def match(separator, subcls, string, ignore_empty=True):
```

Comment:
I know it was my suggestion but I'm still not happy with the naming (on the grounds that, coming back to this review now, it's taken me quite a while to understand what's going on). I therefore suggest we re-name it again to `allow_empty_entries` or maybe (sort of) back to `keep_empty_entries`. Maybe it would help to write down its purpose in the docstring first (i.e. why is this flag needed?) and then see what suits?

Reply:
OK, I've done that and come up with 'match_empty_entries' which makes sense to me. I've also added a simple example to the docstring.